internal/infrastructure/mongo/mongodoc/scene_alignsystem.go

```diff
@@ -0,0 +1,129 @@
+}
+
+type WidgetArea struct {
+	WidgetIds []string
+	Align     string
+}
+
+type WidgetSection struct {
+	Top    WidgetArea
+	Middle WidgetArea
+	Bottom WidgetArea
+}
+
+type WidgetZone struct {
+	Left   WidgetSection
+	Center WidgetSection
+	Right  WidgetSection
+}
+
+type SceneAlignSystemDocument struct {
+	Inner WidgetZone
+	Outer WidgetZone
+}
+
+func NewWidgetAlignSystem(was scene.WidgetAlignSystem) *SceneAlignSystemDocument {
+	widgetAlignDoc := SceneAlignSystemDocument{Inner: buildWidgetZone(&was), Outer: buildWidgetZone(&was)}
+	return &widgetAlignDoc
+}
+
+func (*SceneAlignSystemDocument) ToModelAlignSystem(d SceneAlignSystemDocument) (*scene.WidgetAlignSystem, error) {
+	was := scene.NewWidgetAlignSystem()
+	// Inner Left
+	was.WidgetAreaFrom(toStruct(d.Inner.Left.Top.WidgetIds), d.Inner.Left.Top.Align, "inner", "left", "top")
+	was.WidgetAreaFrom(toStruct(d.Inner.Left.Middle.WidgetIds), d.Inner.Left.Middle.Align, "inner", "left", "middle")
+	was.WidgetAreaFrom(toStruct(d.Inner.Left.Bottom.WidgetIds), d.Inner.Left.Bottom.Align, "inner", "left", "bottom")
+	// Inner Center
+	was.WidgetAreaFrom(toStruct(d.Inner.Center.Top.WidgetIds), d.Inner.Center.Top.Align, "inner", "center", "top")
+	was.WidgetAreaFrom(toStruct(d.Inner.Center.Middle.WidgetIds), d.Inner.Center.Middle.Align, "inner", "center", "middle")
+	was.WidgetAreaFrom(toStruct(d.Inner.Center.Bottom.WidgetIds), d.Inner.Center.Bottom.Align, "inner", "center", "bottom")
+	// Inner Right
+	was.WidgetAreaFrom(toStruct(d.Inner.Right.Top.WidgetIds), d.Inner.Right.Top.Align, "inner", "right", "top")
+	was.WidgetAreaFrom(toStruct(d.Inner.Right.Middle.WidgetIds), d.Inner.Right.Middle.Align, "inner", "right", "middle")
+	was.WidgetAreaFrom(toStruct(d.Inner.Right.Bottom.WidgetIds), d.Inner.Right.Bottom.Align, "inner", "right", "bottom")
+	// Outer Left
+	was.WidgetAreaFrom(toStruct(d.Outer.Left.Top.WidgetIds), d.Outer.Left.Top.Align, "outer", "left", "top")
+	was.WidgetAreaFrom(toStruct(d.Outer.Left.Middle.WidgetIds), d.Outer.Left.Middle.Align, "outer", "left", "middle")
+	was.WidgetAreaFrom(toStruct(d.Outer.Left.Bottom.WidgetIds), d.Outer.Left.Bottom.Align, "outer", "left", "bottom")
+	// Outer Center
+	was.WidgetAreaFrom(toStruct(d.Outer.Center.Top.WidgetIds), d.Outer.Center.Top.Align, "outer", "center", "top")
+	was.WidgetAreaFrom(toStruct(d.Outer.Center.Middle.WidgetIds), d.Outer.Center.Middle.Align, "outer", "center", "middle")
+	was.WidgetAreaFrom(toStruct(d.Outer.Center.Bottom.WidgetIds), d.Outer.Center.Bottom.Align, "outer", "center", "bottom")
+	// Outer Right
+	was.WidgetAreaFrom(toStruct(d.Outer.Right.Top.WidgetIds), d.Outer.Right.Top.Align, "outer", "right", "top")
+	was.WidgetAreaFrom(toStruct(d.Outer.Right.Middle.WidgetIds), d.Outer.Right.Middle.Align, "outer", "right", "middle")
+	was.WidgetAreaFrom(toStruct(d.Outer.Right.Bottom.WidgetIds), d.Outer.Right.Bottom.Align, "outer", "right", "bottom")
+	return was, nil
+}
+
+func toString(wids []*id.WidgetID) []string {
```

Comment:
```suggestion
func widgetIDsToStrings(wids []*id.WidgetID) []string {
```

The name is too abstract.